assembly-main/src/assembly/bin/che.sh

```diff
@@ -632,14 +669,33 @@ function launch_che_server {
 
     fi
 
-    echo -e "Docker container named ${GREEN}che${NC} successfully started."
-    echo -e "Launching Che app server inside the container named ${GREEN}che${NC}."
+    echo -e "Docker container named ${GREEN}${CONTAINER}${NC} successfully started."
+    echo -e "Launching Che in a container named ${GREEN}${CONTAINER}${NC}..."
 
     # Launching tomcat with the start option in a docker container does not run successfully
     # because main process of container exits after starting Che as child process
-    set -x
-    "${DOCKER}" exec -it che bash -c 'true && sudo restart docker && export CHE_HOME=/home/user/che && /home/user/che/bin/che.sh '-p:${CHE_PORT}' --skip:client run' || DOCKER_EXIT=$? || true    
-    set +x
+
+    if [[ "${USE_DEBUG}" == true ]]; then
+   	  DEBUG_PRINT_VALUE=--debug
+	else
+	  DEBUG_PRINT_VALUE=
+	fi
+
+	if ${USE_DEBUG}; then
+	  set -x
+	fi
+
+    "${DOCKER}" exec -i ${CONTAINER} bash -c "sudo rm -rf /home/user/che/lib-copy/* && "`
+                                            `"mkdir -p /home/user/che/lib-copy/ && "`
+                                            `"sudo chown -R user:user /home/user && "`
+                                            `"cp -rf /home/user/che/lib/* /home/user/che/lib-copy && "`
+                                            `"sudo sed -i 's/random/urandom/g' /opt/jre1.8.0_65/lib/security/java.security && "`
```

Comment:
Looks like this change decreses security of random generator. @skabashnyuk WDYT is it ok to accept that change?